src/etc/inc/interfaces.inc

```diff
@@ -6700,6 +6701,37 @@ function set_interface_mtu($interface, $mtu) {
 		}
 	} else {
 		pfSense_interface_mtu($interface, $mtu);
+		set_ipv6routes_mtu($interface, $mtu);
+	}
+}
+
+function set_ipv6routes_mtu($interface, $mtu) {
```

Comment:
This function probably belongs in `/etc/inc/util.inc` instead of here, since it's not technically an interface function. Move it after `function route_add_or_change()` near line 2483

Reply:
done